dashboards-observability/public/components/explorer/sidebar/field.tsx

```diff
@@ -87,8 +107,9 @@ export const Field = (props: IFieldProps) => {
               data-test-subj={`fieldToggle-${field.name}`}
               aria-label={ selected ? removeLabelAria : addLabelAria }
             />
-          ) : <></>
+          ) : null
```

Comment:
Question: why did we replace the short syntax `<>` with null? Is there a reason to do it this way?